integration/examples/annotated-skaffold.yaml

```diff
@@ -194,7 +194,13 @@ deploy:
     #   apply: [""]
     #   delete: [""]
 
- # helm:
+ # helm:    
+    # helm can be passed additional option flags either 
+    # on every command (Global), on install (Install) or on update (Update).
```

Comment:
`update` should say `upgrade`

Reply:
Fixed with https://github.com/GoogleContainerTools/skaffold/pull/1673/commits/52e919999c5485eb8ffda07e896df83970328b87